# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  shoulder_pan_joint:
    has_velocity_limits: true
    max_velocity: 0.25
    has_acceleration_limits: true
    max_acceleration: 0.25
  shoulder_lift_joint:
    has_velocity_limits: true
    max_velocity: 0.3
    has_acceleration_limits: true
    max_acceleration: 0.3
  elbow_joint:
    has_velocity_limits: true
    max_velocity: 0.4
    has_acceleration_limits: true
    max_acceleration: 0.4
  wrist_1_joint:
    has_velocity_limits: false
    max_velocity: 0.5
    has_acceleration_limits: false
    max_acceleration: 0.5
  wrist_2_joint:
    has_velocity_limits: false
    max_velocity: 2.0
    has_acceleration_limits: false
    max_acceleration: 2.0
  wrist_3_joint:
    has_velocity_limits: false
    max_velocity: 3.0
    has_acceleration_limits: false
    max_acceleration: 3.0